src/components/Typography/Heading/__tests__/Heading.spec.jsx

```diff
@@ -20,12 +20,28 @@ describe('Heading', () => {
     return heading.find(heading.props().level)
   }
 
+  const doShallow = (overrides = {}) => {
+    const heading = shallow(
+      <Heading {...defaultProps} {...overrides}>
+        The content
+      </Heading>
+    )
+
+    return heading
+  }
+
   it('renders', () => {
     const heading = render(<Heading level="h2">The heading</Heading>)
 
     expect(heading).toMatchSnapshot()
   })
 
+  it('renders mobile styles by default', () => {
```

Comment:
I think since we have a "Heading.ssr.spec.jsx" file, then this test is not necessary.